### ISSUE TITLE
Diffs no longer working in >=0.6.11

### ISSUE THREAD

**Issue by MichaelBurgess** [amweiss/angular-diff-match-patch#48]
Hi,

After updating from 0.6.10 to 0.6.13 our diffs stopped working. The component just shows the right-obj with no diffs.

We're using the component with Bower in a basic way:

`<pre line-diff right-obj="$ctrl.right()" left-obj="$ctrl.left()" options="$ctrl.options"></pre>`

```
options:
  interLineDiff: true
  ignoreTrailingNewLines: true
```

Do we need to update anything else to get this working in the newer releases?

Thanks,
Mike.

**Comment by amweiss** [comment]
I don't currently use the Bower method do I might need some help setting up. Can you provide any console errors you see while using it? The major items I think that have changed are some of the global variables from the package file. 

The only change of significance is c1089da6a7338ca81e2435d92d53ead8ac86403b

So if you want to fix on the old version for now it should work, then I'll need some help debugging the Bower part.

**Comment by amweiss** [comment]
@MichaelBurgess I'm terribly sorry, somehow the tests were not working. I'll revert the bad commit and push a new version up.

**Comment by amweiss** [comment]
Fixed in 0.6.14 with 4aa231a81f1cb6aebba3534dfe8571f217395910

**Comment by MichaelBurgess** [comment]
Hi @amweiss - thanks for the quick workaround. Have tested `0.6.14` and looks good now. Thanks!
